### PR TITLE
Plasmaman Captain and Security/Brigphys Quality of Life

### PR DESCRIPTION
**Pull request by BrumalLake**
## About The Pull Request
Captains and security/brigphys plasmamen start with their sechailers on and their suit sensors maxed

## Why It's Good For The Game
Less upkeep on things that these jobs should take for granted

## Changelog
:cl:
add: Captain and sec/brigphys plasmamen start with sechailer on
fix: Sensors start maxed on cap and sec/brigphys envirosuits
/:cl:

### CODE REVIEW

File: code/modules/clothing/outfits/plasmaman.dm [hunk 1]
```diff
@@ -143,8 +147,9 @@
 	uniform = /obj/item/clothing/under/plasmaman/atmospherics
 
 /datum/outfit/plasmaman/command
-	name = "Command Plasmamand"
-	
+	name = "Command Plasmaman"
+
+	mask = /obj/item/clothing/mask/gas/sechailer
```
Comment (francinum) [inline code/modules/clothing/outfits/plasmaman.dm:152]: Why do they get a sechailer?

Comment (BrumalLake) [reply code/modules/clothing/outfits/plasmaman.dm:152]: he has one in his suit storage which most captains wear, and hes gotta wear an internals mask anyways

File: code/modules/clothing/under/jobs/Plasmaman/civilian_service.dm [hunk 2]
```diff
@@ -75,6 +75,8 @@
 	icon_state = "command_envirosuit"
 	item_state = "command_envirosuit"
 	item_color = "command_envirosuit"
+	sensor_mode = SENSOR_COORDS
```
Comment (Archanial) [inline code/modules/clothing/under/jobs/Plasmaman/civilian_service.dm:78]: Do other (non plasmeme) variants also start with `SENSOR_COORDS`?

Comment (BrumalLake) [reply code/modules/clothing/under/jobs/Plasmaman/civilian_service.dm:78]: yes, the captain definitely has his sensors start maxed
image

**Review by Archanial — APPROVED** [review]

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**